extensions/interactions/ItemSelectionInput/validator.js

```diff
@@ -89,14 +89,30 @@ oppia.filter('oppiaInteractiveItemSelectionInputValidator', [
         baseInteractionValidationService.getAnswerGroupWarnings(
           answerGroups, stateName));
 
-      if (!defaultOutcome ||
-          $filter('isOutcomeConfusing')(defaultOutcome, stateName)) {
-        warningsList.push({
-          type: WARNING_TYPES.ERROR,
-          message: (
-            'Please clarify the default outcome so it is less confusing to ' +
-            'the user.')
-        });
+      var selectedChoices = [];
+      for (var i = 0; i < answerGroups.length; i++) {
```

Comment:
Similar considerations apply here.

There may actually be opportunity to add additional validation, too. For example, suppose the maximum number of choices that a student may pick is 1. If someone added a rule that said "is equal to X", where X contained 2 or more choices, that rule would never trigger, and we could show them a warning that there's a superfluous rule. Handling this within the scope of this PR is optional; just thought I'd point it out.